deploy-manage/monitor/autoops-vs-stack-monitoring.md

```diff
@@ -26,18 +31,24 @@ With Stack Monitoring, you are responsible for storing your monitoring data. Thi
 ### Setup
 
 #### AutoOps [ao-setup]
-On {{ech}} (ECH), AutoOps is set up and enabled automatically in all supported [regions](/deploy-manage/monitor/autoops/ec-autoops-regions.md), with no action required from you.
-
-:::{note}
-AutoOps is planned to be available in self-managed environments in the future.
-:::
+On {{ech}} and {{serverless-full}}, AutoOps is set up and enabled automatically in all supported [regions](/deploy-manage/monitor/autoops/ec-autoops-regions.md), with no action required from you.
 
 :::{image} /deploy-manage/images/cloud-autoops-setup.png
 :alt: Diagram showing AutoOps setup in Elastic Cloud
 :::
 
+For {{ece}} (ECE), {{eck}} (ECK), and self-managed clusters, you need to go through a [connection process](/deploy-manage/monitor/autoops/cc-connect-self-managed-to-autoops.md) to connect to AutoOps. This involves installing {{agent}} to ship your monitoring metrics to {{ecloud}} through [](/deploy-manage/cloud-connect.md). Since you can only access AutoOps from the {{ecloud}} UI, you need an {{ecloud}} account for this service.
```

Comment:
```suggestion
For {{ece}} (ECE), {{eck}} (ECK), and self-managed clusters, you need to [connect your cluster to AutoOps](/deploy-manage/monitor/autoops/cc-connect-self-managed-to-autoops.md). This involves installing {{agent}} to ship your monitoring metrics to {{ecloud}} through [](/deploy-manage/cloud-connect.md). Since you can only access AutoOps from the {{ecloud}} UI, you need an {{ecloud}} account for this service.
```
Just rephrasing a bit to avoid the duplicate "connect" in "go through a connection process to connect".